src/gromacs/nblib/simulationstate.h

```diff
@@ -58,34 +58,44 @@
 namespace nblib
 {
 
-//! Simulation state description that serves as a snapshot of the system
-//! being analysed. Needed to init an MD program. Allows hot-starting simulations.
+/*! \libinternal
+ * \ingroup nblib
+ * \brief Simulation State
+ *
+ * Simulation state description that serves as a snapshot of the system
+ * being analysed. Needed to init an MD program. Allows hot-starting simulations.
+ */
+
```

Comment:
I think we don't want spaces between the doxygen and the class it documents.

Reply:
Done

Reply:
It was still the same but I uploaded a small fix.